### PR TITLE
Create LCMS_Visualization.js

### PR DESCRIPTION
**Pull request by rcr-usfs**

### CODE REVIEW

**Comment by gino-m** [comment]
@rcr-usfs Thanks for resubmitting this with the commit ownership fix. LGTM. Over to @simonff for approval.

**Comment by jdbcode** [comment]
Here is the script with recommended format changes (spaces between arguments, 80 char line length, section breaks/new lines, etc)

https://code.earthengine.google.com/e59241ff8c733d3548cdbae092651246

@rcr-usfs can you please review the linked script above and replace the current version in this PR with it, if you are okay with the changes.

**Review by rcr-usfs — COMMENTED** [review]
This looks mostly good! I made a couple minor changes: https://code.earthengine.google.com/026394a9019ccdcd21aa39c14c6e409f

**Comment by jdbcode** [comment]
Thanks for making the changes -- looks good to me.

@simonff please review and mark as approved to trigger the merging process.

File: datasets/scripts/LCMS_Visualization.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  });
+}
+
+
+// #############################################################################
+// ### Bring in LCMS annual outputs ###
+// #############################################################################
+
+var lcms = ee.ImageCollection(
+  'projects/lcms-292214/assets/Final_Outputs/2020-5/LCMS');
```
Comment (simonff) [inline datasets/scripts/LCMS_Visualization.js:140]: please use the catalog collection name USFS/GTAC/LCMS/v2020-5

Comment (rcr-usfs) [reply datasets/scripts/LCMS_Visualization.js]: I updated the collection path. @simonff  please review.

**Review by simonff — APPROVED** [review]

**Pull request merged**